pants

```diff
@@ -49,7 +49,7 @@ function exec_pants_bare() {
   # Because the venv has been activated, we simply say `python` and it will use the venv's version. We cannot use
   # `${PY}` because it could use the wrong interpreter if the value is an absolute path.
   PYTHONPATH="${PANTS_SRCPATH}:${PYTHONPATH}" RUNNING_PANTS_FROM_SOURCES=1 \
-    exec python "${PANTS_EXE}" "$@"
+    exec ${PANTS_PREPEND_ARGS:-} "$(venv_dir)/bin/python" "${PANTS_EXE}" "$@"
```

Comment:
does it need quotes?

Reply:
No, because using quotes would make the shell see an argument but one that is empty, whereas without the quotes, the shell won't see any argument when that variable is empty. The shell would then try to run a command with an empty name:

```
$ "" ls
-bash: : command not found
```

Reply:
Hmm that comment above the exex is not out of date.

Reply:
Pushed a commit to remove that comment as moot.